src/main/java/santiagoAndFerdy/vgs/rmi/RmiServer.java

```diff
@@ -26,4 +27,27 @@ public void register(String url, Remote impl) throws MalformedURLException, Remo
         Naming.rebind(url, impl);
         System.out.println("Registered object at " + url);
     }
+    public void unRegister(String url){
```

Comment:
You can make register return something that can be used to unregister. Would make usage a little easier.

Make sure that unregistering is useful first. It seems like Naming only facilitates resolving. Once you already have a proxy you can still make remote calls.